### DISCUSSION TITLE
Prevent sliding multiple slides by swiping on mobile phone

### DISCUSSION THREAD

**Discussion by TheZoker** [nolimits4web/swiper#4239] [Q&A] [answered]
Hi there,

I use the following config for my slider:
```javascript
const slider = new Swiper('.swiper-container', {
    slidesPerView: 1,
    spaceBetween: 0,
    centeredSlides: true,
    cssMode: true,
    navigation: {
        nextEl: '.swiper-button-next',
        prevEl: '.swiper-button-prev',
    }
});
```

Now when a user swipes on a mobile phone, the slider jumps trough multiple slides, based on how intense the user swipes. 
How can I disable this and allow only to jump one slide with one swipe? So no matter how intense a user swipes, he only swipes one slide. 

Thanks!

**Comment by nolimits4web** [comment, marked as answer]
It is because you use `cssMode: true`. In this mode swiper basically uses `overflow: 'auto'` and no way to force swiping to fixed slide

**Reply by TheZoker** [reply]
Ah I see, I removed the `cssMode: true` and now it works. Thanks! 🎉
